### PR TITLE
(PC-13383) fix(A11y): added hyperlinks on client navigation

### PR DESCRIPTION
**Pull request by kopax-polyconseil**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-13383

## Checklist

I have:

- [x] Made sure the title of my PR follows the [convention](1) `($jira) $type($scope): $summary`.
- [x] Made sure my feature is working on the relevant real / virtual devices (native and web).
- [x] Written **unit tests** native (and web when implementation is different) for my feature.
- [ ] Added a **screenshot** for UI tickets.
- [ ] Attached a **ticket number** or **github dev name** for any added TODO / FIXME.
- [ ] Added new reusable components to **AppComponents** page and communicate to the team on slack.
- [ ] If my PR is a bugfix, I add the link of the "résolution de problème sur le bug" [on Notion](2)
- [ ] Made sure translations file is up to date (`yarn translations:extract`)

## Screenshots

| iOS | Android | Mobile - Chrome | Desktop - Chrome | Desktop - Safari |
| --: | ------: | --------------: | ---------------: | ---------------: |
|     |         |                 |                  |                  |

[1]: https://github.com/pass-culture/pass-culture-app-native/blob/master/doc/standards/pr-title.md
[2]: https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a

### CODE REVIEW

File: src/ui/web/link/Link.web.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
+/**
+ * Ce fichier est copier de @react-navigation/native et rajoute un hack nécessaire pour react-native-web,
+ * En effet, nous n'avons pas trouvé comment annuler l'événement qui reload la page au click sur l'anchor tag en ReactJS,
+ * Nous attachons donc un événement en vanillaJS pour annuler l'événement pour le Web
+ *
+ * Related issues :
+ * - https://github.com/react-navigation/react-navigation/issues/10295
+ * - https://github.com/necolas/react-native-web/issues/2206
+ */
```
Comment (Lucasbeneston) [inline src/ui/web/link/Link.web.tsx:9]: C'est pas grave de mettre du texte en français ?

File: src/ui/web/link/types.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
+/**
+ * Ce fichier contient les types copier de @react-navigation/native Link
+ *
+ * Related issues :
+ * - https://github.com/react-navigation/react-navigation/issues/10295
+ * - https://github.com/necolas/react-native-web/issues/2206
+ */
```
Comment (Lucasbeneston) [inline src/ui/web/link/types.ts:7]: idem

**Review by Lucasbeneston — APPROVED** [review]
👍

File: src/ui/web/link/Link.web.tsx (hunk 1 above)
Comment (kopax-polyconseil) [reply src/ui/web/link/Link.web.tsx:9]: C'est pour nous, pas gênant, c'est retiré du code, mais avec une convention ca serait mieux

**Pull request merged**